sros/docker/launch.py

```diff
@@ -151,17 +151,65 @@ def line_card_config(
             }
         ],
     },
+    ## IXR-X1
+    ## cpu=2 ram=4 slot=A chassis=ixr-x card=cpm-ixr-x/imm32-qsfp28+4-qsfpdd
     "ixr-x1": {
         "deployment_model": "distributed",
-        "min_ram": 4,  # minimum RAM requirements
+        # control plane (CPM)
         "max_nics": 36,
-        **line_card_config(
-            chassis="ixr-x",
-            card="cpm-ixr-x",
-            card_type="imm32-qsfp28+4-qsfpdd",
-            mda="m32-qsfp28+4-qsfpdd",
-            integrated=False,
-        ),
+        "cp": {
+            "min_ram": 4,
+            "timos_line": "slot=A chassis=ixr-x card=cpm-ixr-x",
+        },
+        # line card (IOM/XCM)
+        "lcs": [
+            {
+                "min_ram": 4,
+                "timos_line": "chassis=ixr-x slot=1 card=imm32-qsfp28+4-qsfpdd",
+                "card_config": """
+            """,
+            }
+        ],
+    },
+    ## IXR-X3
+    ## cpu=2 ram=4 slot=A chassis=ixr-x card=cpm-ixr-x/imm36-qsfpdd
+    "ixr-x3": {
+        "deployment_model": "distributed",
+        # control plane (CPM)
+        "max_nics": 36,
+        "cp": {
+            "min_ram": 4,
+            "timos_line": "slot=A chassis=ixr-x card=cpm-ixr-x",
+        },
+        # line card (IOM/XCM)
+        "lcs": [
+            {
+                "min_ram": 4,
+                "timos_line": "chassis=ixr-x slot=1 card=imm36-qsfpdd",
+                "card_config": """
+            """,
```

Comment:
@sdktr you don't need to configure the cards for x?

Reply:
thanks for pointing out the probable cause for my deployment failing. Hold on